### PR TITLE
Fix originInfo eventType differs from date type test.

### LINKED ISSUE #1932: Fix "originInfo eventType differs from date type" test
From `origin_info_spec.rb`.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1932

## Why was this change made?
Broken test bad.

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000):
  Success:   98117 (98.117%)
  Different: 1334 (1.334%)
  To Cocina error:     18 (0.018%)
  To Fedora error:     0 (0.0%)
  Missing:     531 (0.531%)
```

After:
```
Status (n=100000):
  Success:   98153 (98.153%)
  Different: 1298 (1.298%)
  To Cocina error:     18 (0.018%)
  To Fedora error:     0 (0.0%)
  Missing:     531 (0.531%)
```

File: app/services/cocina/mods_normalizer.rb [hunk 1]
```diff
@@ -204,23 +208,24 @@ def normalize_origin_info_event_types
...
+        add_event_type('publication', origin_info_node) && next if frequency.present? && origin_info_node['eventType'].nil?
       end
     end
     # rubocop:enable Metrics/CyclomaticComplexity
     # rubocop:enable Metrics/PerceivedComplexity
+    # rubocop:enable Metrics/AbcSize
 
     def add_event_type(value, origin_info_node)
-      origin_info_node['eventType'] = value if origin_info_node[:eventType].blank?
+      origin_info_node['eventType'] = value
```
Comment (jcoyne) [inline app/services/cocina/mods_normalizer.rb:228]: Since there's no longer a conditional here, would it make sense to just inline that in the method above?  I found this really hard to follow because you were testing `origin_info_node['eventType'].nil?` and then calling a separate method that was mutating that value.

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/services/cocina/mods_normalizer.rb (hunk 1 above)
Comment (justinlittman) [reply app/services/cocina/mods_normalizer.rb]: I tried inlining, but it just made things worse. So I took a different tact -- encapsulate more of the logic in the helper method to reduce the complexity of the main method.

**Review by jcoyne — APPROVED** [review]
Thanks, this is much easier for me to read.
